tests/pdarray_creation_test.py

```diff
@@ -62,23 +61,34 @@ def test_bigint_creation(self):
             self.assertIsInstance(pda, ak.pdarray)
             self.assertEqual(10, len(pda))
             self.assertEqual(ak.bigint, pda.dtype)
+            self.assertEqual(pda[-1], bi + 9)
 
         # test array and arange infer dtype
         arr = ak.array([bi, bi + 1, bi + 2, bi + 3, bi + 4])
         ara = ak.arange(bi, bi + 5)
+        # TODO add .to_list and .to_ndarray functionality
+        for i in range(5):
+            self.assertEqual(arr[i], bi + i)
+            self.assertEqual(ara[i], bi + i)
 
         # test that max_bits being set results in a mod
         pda = ak.array([bi, bi + 1, bi + 2, bi + 3, bi + 4], max_bits=200)
+        for i in range(5):
+            self.assertEqual(i, pda[i])
```

Comment:
Should this be self.asserEqual(bi + i, pda[i])? Does bi roll back down to 0 in this case?

Reply:
ah so the reason why it rolls over in this case is because `max_bits = 200`, so  since `bi = 2**200` is equivalent to `0`

Reply:
Perfect, thank you